src/dra/7E4BC.c

```diff
@@ -1217,7 +1217,260 @@ void EntityPlayerPinkEffect(Entity* self) {
 }
 
 // player dissolves into pixels
-INCLUDE_ASM("dra/nonmatchings/7E4BC", EntityPlayerDissolves);
+void EntityPlayerDissolves(Entity* self) {
+    const int PrimCount = 36;
+    const int Iterations = 40;
+    u8 sp4F;
+    u8 sp4E;
+    u8 sp4D;
+    u8 sp4C;
+    s16 sp4A;
+    s16 sp48;
+    s16 sp46;
+    s16 sp44;
+    s16 sp42;
+    s32 sp3C;
+    s16* sp38;
+    Primitive* prim;
+    s32 temp_i;
```

Comment:
```suggestion
    s32 i;
```
let's just rename this one as `i`